webpack.config.js

```diff
@@ -3,20 +3,43 @@ const path = require('path');
 const TerserWebpackPlugin = require("terser-webpack-plugin");
 const OptimizeCssAssetsPlugin = require("optimize-css-assets-webpack-plugin");
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 const FileManagerPlugin = require('filemanager-webpack-plugin');
+const dotenv = require('dotenv');
 
 module.exports = function(_env, argv) {
   const isProduction = argv.mode === "production";
   const isDevelopment = !isProduction;
   /*
    * output to static file for ease of development
    */
-  const outputDirectory = isDevelopment?"/dashboard/static":"/dashboard/dist";
+  const outputDirectory = isDevelopment?"/patientsearch/static":"/patientsearch/dist";
   const jsDirectory = `${outputDirectory}/js`;
   const templateDirectory = `${outputDirectory}/templates`;
+  /*
+   * output to static file for ease of development
+   */
+  const envConfig = dotenv.config();
+  // call dotenv and it will return an Object with a parsed key 
+  const env = envConfig ? envConfig.parsed : null;
+  let envKeys = {};
+
+  if (env) {
+    // reduce it to a object
+    try {
+      envKeys = Object.keys(env).reduce((prev, next) => {
+        prev[`process.env.${next}`] = JSON.stringify(env[next]);
+        return prev;
+      }, {});
+    } catch(e) {
+      console.log("Error occurred processing .env config ", e);
```

Comment:
I couldn't find a `.env` file in the repo, but wanted to caution that docker-compose also uses a `.env` file and can lead to some confusing results

Reply:
thank you, ivan. I did just remove this code

Reply:
we got rid of that.  accidental oversight.